test_flickr.py

```diff
@@ -18,35 +25,32 @@ def test_flickr_get_photos_list_correct():
         assert photo["secret"] is not None
 
 
+def get_photo_correct_mock_request(*args, **kwargs):
+    m = mock.MagicMock()
+    m.content = b"/ff/ff"
+    return m
+
+
 def test_flickr_get_photo_correct():
     """
     check that returned value exist and have types bytes and string
     """
-    flickr = Flickr()
+    flickr = Flickr(get_photo_correct_mock_request)
     test_flickr_get_photo_result_binary, test_flickr_get_photo_result_name = flickr.get_photo(
         {"farm": "5", "server": "4504", "id": "24003882568", "secret": "ca14f88bec", "title": "test"})
     assert test_flickr_get_photo_result_binary is not None
     assert test_flickr_get_photo_result_name is not None
     assert isinstance(test_flickr_get_photo_result_binary, bytes)
     assert isinstance(test_flickr_get_photo_result_name, str)
 
+
+def get_photo_incorrect_mock_request(*args, **kwargs):
+    raise ValueError
+
+
 def test_flickr_get_photo_incorrect():
```

Comment:
The test looks completely meaningless with this mock. Actual Flickr() implementation doesn't participate in the test at all.